dragonfly/implementations/at_operator.py

```diff
@@ -0,0 +1,472 @@
+from __future__ import absolute_import
+
+try:
+    import dateutil, funcsigs
+    # for google
+    import googleapiclient.discovery
+    from google.oauth2.credentials import Credentials
+except ImportError:
+    pass
+
+import datetime, logging, os, time, json
+# for slack
+import slackclient
+
+from dripline.core import Endpoint, fancy_doc
+from .subprocess_mixin import *
```

Comment:
do we use anything other than SlowSubprocessMixin from here? is an explicit import better?